cdc/metrics/grafana/tikv-cdc.json

```
@@ -1225,7 +1225,7 @@
           ],
           "timeFrom": null,
           "timeShift": null,
-          "title": "Table count maintained by owner",
+          "title": "KeySpan count maintained by owner",
           "transform": "timeseries_aggregations",
```

```suggestion
          "title": "Keyspan count maintained by owner",
```

Suggest to use uniform case of name.
Rest LGTM.

---

fixed, thanks~